src/renderer/components/ResourcesLinks/index.jsx

```diff
@@ -60,9 +62,11 @@ export default function ResourcesTab(props) {
   return (
     <React.Fragment>
       <a href={userGuideURL} onClick={handleClick}>
-        {"User's Guide"}
+        <MdOpenInNew className="mr-1" />

```

Comment:
I didn't see `mr-1` added to the CSS. Is this defined somewhere else?

Reply:
It's defined for us in bootstrap